config/locales/server.en.yml

```diff
@@ -1881,6 +1881,7 @@ en:
     log_mail_processing_failures: "Log all email processing failures to <a href='%{base_path}/logs' target='_blank'>/logs</a>"
     email_in: 'Allow users to post new topics via email (requires manual or pop3 polling). Configure the addresses in the "Settings" tab of each category.'
     email_in_min_trust: "The minimum trust level a user needs to have to be allowed to post new topics via email."
+    email_in_authserv_id: "The authentication service identifier used by the configured polling method."
```

Comment:
I think the setting description could be improved. Without looking at the code I have no idea what that setting is for and how to use it.